products-service/config/index.ts

```diff
@@ -0,0 +1,20 @@
+import dotenv from 'dotenv';
+import { ClientConfig } from "pg";
+
+dotenv.config({ path: './config/config.env' });
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
```

Comment:
I don't see environment variabless maping in serverless file.
When you run it locally you use dotenv plugin and local .env file, but how hosted lambda got this variables?